### PR TITLE
Remove hostoperatorconfig and registrationservice CRDs

### PR DESCRIPTION
**Pull request by rajivnathan**
Related PR: https://github.com/codeready-toolchain/api/pull/256

- Generated file changes for removal of hostoperatorconfig and registrationservice CRDs
- Updated unit tests

Related issue: https://issues.redhat.com/browse/CRT-1176

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]

**Review by MatousJobanek — COMMENTED** [review]
You need to update all files that contain a reference to the CRDs.
* CSV in ./bundle/manifests/toolchain-host-operator.clusterserviceversion.yaml
* config/crd/kustomization.yaml
* ./PROJECT

**Comment by rajivnathan** [comment]
@MatousJobanek I updated `config/crd/kustomization.yaml` and `./PROJECT` but the files in `./bundle` are not git ignored. However, I updated `config/manifests/bases/host-operator.clusterserviceversion.yaml`. I guess the files in `./bundle` will be regenerated so it shouldn't be a problem?

**Comment by MatousJobanek** [comment]
> but the files in ./bundle are not git ignored. However, I updated config/manifests/bases/host-operator.clusterserviceversion.yaml. I guess the files in ./bundle will be regenerated so it shouldn't be a problem?

yes, you are right - I copy-pasted the wrong line. Sorry for confusions.

File: go.mod [hunk 1]
```diff
@@ -29,4 +29,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/rajivnathan/api v0.0.0-20210903005117-9931df68c579
```
Comment (MatousJobanek) [inline go.mod:32]: Reminder :-)

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**